shopinvader/wizards/shopinvader_partner_binding_line.py

```diff
@@ -54,15 +55,15 @@ def action_apply(self):
                 lambda x, b=backend: x.backend_id == b
             )
             if not partner_binding:
-                bind_values = {"backend_id": backend.id}
-                partner_values = {
-                    "shopinvader_bind_ids": [(0, False, bind_values)]
+
+                bind_values = {
+                    "backend_id": backend.id,
+                    "record_id": record.partner_id.id,
                 }
-                # Locomotive doesn't work with uppercase.
```

Comment:
why do you remove this comment ?
I think it makes sense to keep it

Reply:
Ooops, wrong copy paste. Done